### PR TITLE
Change conf.example.json directory to correct one

### PR DESCRIPTION
**Pull request by nuokh**
Looks like the folder /dist/ in taiga-front got renamed to /conf/.

### CODE REVIEW

**Comment by Justman10000** [comment]
@bameda

File: setup-production.adoc [hunk 1]
```diff
@@ -358,14 +358,14 @@ Map the file into `/taiga-back/settings/config.py`. Have in mind that you have t
 
 .Map a `conf.json` file
 
-From https://github.com/kaleidos-ventures/taiga-front[taiga-front] download the file `dist/conf.example.json` and rename it:
+From https://github.com/kaleidos-ventures/taiga-front[taiga-front] download the file `conf/conf.example.json` and rename it:
```
Comment (yamila-moreno) [inline setup-production.adoc:361]: good catch, thanks!

File: setup-production.adoc [hunk 2]
```diff
@@ -358,14 +358,14 @@ Map the file into `/taiga-back/settings/config.py`. Have in mind that you have t
...
 
 [source,bash]
 ----
-mv dist/conf.example.json dist/conf.json
+mv conf/conf.example.json conf/conf.json
```
Comment (yamila-moreno) [inline setup-production.adoc:365]: this change is ok, given the original code, but it really should be:
`mv conf.example.json conf.json`

File: setup-production.adoc [hunk 3]
```diff
@@ -358,14 +358,14 @@ Map the file into `/taiga-back/settings/config.py`. Have in mind that you have t
...
 ----
 
-Edit it with your own configuration and map the file into `/taiga-front/dist/config.py`.
+Edit it with your own configuration and map the file into `/taiga-front/conf/config.py`.
```
Comment (yamila-moreno) [inline setup-production.adoc:368]: this sentence is no longer in the current `main` but the documentation bug is still present in the current branch.

This part is about mapping "conf.json" not, "config.py". And, according to [`docker-compose.yml`](https://github.com/kaleidos-ventures/taiga-docker/blob/main/docker-compose.yml#L110) it should be mapped into a different path.

**Review by yamila-moreno — CHANGES_REQUESTED** [review]
Hi @Justman10000 
thank you for your contribution! We're currently preparing a release and doing a clean-up of pull-requests.

Your changes are mostly correct, but currently the repository changed and now there are some conflicts. If you'd like to keep this contribution, you can rebase `main` and remove conflicts.

I've left some changes to be made in order to merge this to main. If you want to continue with this PR, I'd like to help you; otherwise, we can made these fixes.

Cheers!

**Comment by nuokh** [comment]
Thanks for the offer, but I decided not to move further with our Taiga project due to too many issues in our test phase, so I am kind of "out of the loop" :-) Great to hear the project is picking up some steam again, tho!

**Comment by yamila-moreno** [comment]
Hi @nuokh thanks for your answer. I totally understand if you have your focus in a different part. We'll take care of these changes.

And you're welcome in the community whenever you want :)

Cheers!!

**Pull request closed without merging**